tests/test_workers/test_tasks/test_opm_operations.py

```diff
@@ -373,3 +373,41 @@ def test_opm_registry_rm(mock_run_cmd):
     opm_args = mock_run_cmd.call_args[0][0]
     assert opm_args[:3] == ['opm', 'registry', 'rm']
     assert ','.join(packages) in opm_args
+
+
+@pytest.mark.parametrize(
+    'from_index, is_fbc', [('some-fbc-index:latest', True), ('some-sqllite-index:latest', False)]
```

Comment:
Could we parametrize approach with `from_index`, `is_fbc` and `result` option? This way we could avoid using If statements in tests.

Reply:
or add a path to the database as one parameter. This could make it easier

Reply:
Since its an assertion check, I let it be in if else